c/hello/hello.c

```diff
@@ -1 +1,3 @@
-main(){puts("Hello, World!");}
+#include<stdio.h>
+int main()
+{ printf("Hello World");}
```

Comment:
@dchenmei Originally authored this file. Feel free to PR a new file with your new code though!